react/features/subtitles/middleware.js

```diff
@@ -47,23 +51,46 @@ MiddlewareRegistry.register(store => next => action => {
  */
 function _endpointMessageReceived({ dispatch, getState }, next, action) {
     const json = action.json;
+    const translationLanguage
+        = APP.conference._room.getLocalParticipantProperty(
```

Comment:
do we need to use `APP` here? I think the conference is accessible through the store.